src/event.js

```diff
@@ -507,6 +507,41 @@ class ICalEvent {
         return this;
     }
 
+    /**
+     * Set/Get the event's geo
+     *
+     * @param {String|object} [geo]
+     * @since 1.5.0
+     * @returns {ICalEvent|String}
+     */
+    geo(geo) {
+        if (geo === undefined) {
+            if (!this._data.geo) {
+                return null;
+            } else {
+                return this._data.geo.lat + ';' + this._data.geo.lon;
+            }
+        }
+
+        let geoStruct = {};
+        if (typeof geo === 'string') {
+            const geoParts = geo.split(';');
+            geoStruct.lat = parseFloat(geoParts[0]);
+            geoStruct.lon = parseFloat(geoParts[1]);
+        } else {
+            geoStruct = geo;
+        }
+
+        if (!geoStruct || !isFinite(geoStruct.lat) || !isFinite(geoStruct.lon)) {
+            //throw "invalid geo";
```

Comment:
```suggestion
            throw new Error('`geo` isn\'t formated correctly. See https://github.com/sebbo2002/ical-generator#geostringobject-geo');
```

Reply:
Is there a reason for not using native getters/setters previously? It sounds like a good idea to use them in `v2.0.0`.

Reply:
I also thought about it when I built the `1.0.0`. I rejected the idea because I thought that it would have no special advantages except some syntax sugar, but wouldn't be backwards compatible to the previous version. I want to avoid aliases / duplicate code as far as possible.

Reply:
I would hope using native getters and setters would help bring consistency by making it more clear that properties could be reset. I see there's some polish to do in that area in the code, tests, and types.

I didn't want to touch every file in this PR, but I did what was needed for `geo` and rebased, so this PR looks all good-to-go to me.

Reply:
All right, all right. I'll take a closer look at the coming days when I get there. Thank you very much.